README.md

```diff
@@ -19,65 +19,62 @@ DocBlock comments to generate a complete set of API Documentation.
 Inspired by phpDocumentor 1 and JavaDoc it continues to innovate and is up to date
 with the latest technologies and PHP language features.
 
-Features
---------
+phpDocumentor v2 (stable)
+--------------
+The current stable version of phpdocumentor is v2.9. Which can be used to generate documentation for php
+applications up to php version 7.0. phpDocumentor is an old tool that doesn't support all 7.0 introduced features.
+It will not process any scalar type hints and requires docblock tags to generate api documentation.
+If your application/library is using php 7.0 or lower consult our [v2 branch] for installation instructions
+and detailed information about how to use phpDocumentor.
 
-phpDocumentor supports the following:
+phpDocumentor v3 (unstable)
+---------------
 
-* *PHP 7.0 compatible*, full support for Namespaces, Closures and more is provided.
-* *Shows any tag*, some tags add additional functionality to phpDocumentor (such as @link).
-* *Processing speed*, Zend Framework experienced a significant reduction in processing time compared to phpDocumentor 1.
-* *Low memory usage*, peak memory usage for small projects is less than 20MB, medium projects 40MB and large frameworks 100MB.
-* *Incremental parsing*, if you kept the Structure file from a previous run you get an additional performance boost of up
-  to 80% on top of the mentioned processing speed above.
-* *Easy template building*, if you want to make a branding you only have to call 1 task and edit 3 files.
-* *Command-line compatibility with phpDocumentor 1*, phpDocumentor 2 is an application in its own right but the
-  basic phpDocumentor 1 arguments, such as --directory, --file and --target, have been adopted.
-* *Two-step process*, phpDocumentor first generates a cache with your application structure before creating the output.
-  If you'd like you can use that to power your own tools or formatters!
+**Note:**
+The phpDocumentor team is currently working on a new phpDocumentor generation. Which will have full support for the latest features in PHP, so you can take advantage of all native type hints. v3 is still under heavy development
+and is not stable enough to use in a daily development flow. 
 
-*Please note* that phpDocumentor 3 is still under heavy development. We aim to add all features needed to have full support
-for php 7+. But at this moment that is not the case.
+### nightly builds
+
+Phpdocumentor doesn't have a nightly release. But during each pipeline, a [phar] artifact is built. If you want 
```

Comment:
s/release. But/release. However,/

Reply:
Still needs a comma right after `However`-- `However, during each pipeline, `

... unless you want to adjust the clause sequencing -- `However, a [phar] artifact is built during each pipeline.`